### ISSUE TITLE
CenterPoint is not using Sync BN

### ISSUE THREAD

**Issue by haotian-liu** [open-mmlab/mmdetection3d#394]
Hi, I noticed in CenterPoint configs, no sync_bn is used. However, in the [original CenterPoint code base](https://github.com/tianweiy/CenterPoint/blob/76b14c64e6dc7f04633cea576ae8ecd8257a416f/det3d/torchie/apis/train.py#L284), it will convert the batch norms to sync_bn when using distributed training.

Is there a reason for not using the sync bn in your implementation for CenterPoint? Thank you!

**Comment by tianweiy** [comment]
it adds apex dependency I guess. Also it seems not super important (no ablation yet though)

**Comment by haotian-liu** [comment]
Thank you for your clarification!
